host/sgx/loadelf.c

```diff
@@ -797,10 +797,12 @@ static oe_result_t _link_elf_image(
             else
             {
                 if ((symbol->st_info >> 4) != STB_WEAK)
-                    OE_RAISE(
+                    OE_RAISE_MSG(
```

Comment:
What is the difference between `OE_RAISE` and `OE_RAISE_MSG`?

Reply:
`OE_RAISE` does not print message, just raises errors.

Reply:
Ah ok.